architecture/additional_concepts/f5_big_ip.adoc

```diff
@@ -130,6 +130,12 @@ xref:../../install_config/router/f5_router.adoc#setting-up-f5-native-integration
 integration of F5 with {product-title}], you do not need to configure a ramp
 node for F5 to be able to reach the pods on the overlay network as created by
 OpenShift SDN.
+
+Also, only *F5 BIG-IP®* appliance version 12.x and above will work with the native integration
+presented in this section. You also need sdn-services add-on license for the
```

Comment:
Does `sdn-services` need mark-up? The _architecture/additional_concepts/f5_big_ip.adoc_ uses bolding: *sdn-services*.